PresentIT/Models/UserProfileViewModel.cs

```diff
@@ -12,6 +13,6 @@ public class UserProfileViewModel
         public string Name { get; set; }
         public string ProfileImage { get; set; }
         public string UserID { get; set; }
-        public Boolean Role { get; set; }
+        public bool Role { get; set; }
```

Comment:
There isn't really any difference in this other than `bool` is just an alias of `Boolean`, but you'll probably notice `bool` more in c# code apparently 😛